### ISSUE TITLE
Handle multiple validation sets

### ISSUE THREAD

**Issue by lewfish** [azavea/raster-vision#359]
The current config library only handles a single validation set. The validation set is used for training and eval (and the test set is only used for prediction). It would be nice to be able to specify multiple named sets of scenes that can be evaluated individually as part of a single run of eval.

**Comment by lewfish** [comment]
I'm closing issues that were just "nice to haves / ideas"  that have been open for a long time and we don't have any plans to support in the near term.
